docs/rst/user_manual/user_manual_glossary.rst

```diff
@@ -44,49 +44,20 @@ DDS nomenclature
 
 .. glossary::
 
-    DomainParticipant
-        A DomainParticipant is the entry point of the application to a DDS Domain.
-        Every DomainParticipant is linked to a single domain from its creation,
-        and contains all the Entities related to that domain.
-        It also acts as a factory for Publisher, Subscriber and Topic.
-
-        See `Fast DDS documentation <https://fast-dds.docs.eprosima.com/en/v2.4.1/fastdds/dds_layer/domain/domainParticipant/domainParticipant.html>`__
-        for further information.
```

Comment:
You've removed this when moved. I'd suggest to keep the link

Reply:
It was a moving error :+1: